src/script/components/MessagesList/Message/FailedToAddUsersMessage.tsx

```diff
@@ -50,6 +50,9 @@ const errorMessageType = {
   [AddUsersFailureReasons.UNREACHABLE_BACKENDS]: 'OfflineBackend',
 } as const;
 
+// Mobile platforms are not using the full specs yet, this flag allows to switch to MVP specs if product decides to
+const useMvpString = false;
```

Comment:
why adding a flag and not just commenting code out (with the explanation that you added here)?

Reply:
I started out with commenting out the code :upside_down_face: 
Since we need to remove the code anyway, I'll revert to that